components/lwip/Kconfig

```diff
@@ -846,6 +846,23 @@ menu "LWIP"
                 First argument of sntp_setserver/sntp_setservername functions
                 is limited to this value.
 
+        config LWIP_DHCP_GET_NTP_SRV
+            bool "Request NTP servers from DHCP"
+            default n
+            help
+                If enabled, LWIP will add 'NTP' to Parameter-Request Option sent via DHCP-request.
+                DHCP server might reply with an NTP server address in option 42.
+                SNTP callback for such replies should be set accordingly (see sntp_servermode_dhcp() func.)
+
+        config LWIP_DHCP_MAX_NTP_SERVERS
+            int "Maximum number of NTP servers aquired via DHCP"
+            default 1
+            range 1 16
+            depends on LWIP_DHCP_GET_NTP_SRV
+            help
+                Set maximum number of NTP servers aquired via DHCP-offer.
+                Must be less or equal to total "Maximum number of NTP servers".
```

Comment:
Question: Does the lwip throws a compile `#error` if it `> SNTP_MAX_SERVERS` ? maybe we should if it doesn't.

Reply:
No, it does not. Actually if I get this right any extra DHCP-servers would be just silently ignored. Not sure that throwing an error and stop here is a good idea.
@david-cermak I can make a test setup with dhcp-server and confirm this if you like?